test/unit/test_UFInterpolation.cc

```diff
@@ -11,6 +11,12 @@ bool verifyInterpolant(PTRef itp, PartitionManager & pManager, ipartitions_t con
     return VerificationUtils(logic).verifyInterpolantInternal(pManager.getPartition(Amask, PartitionManager::part::A), pManager.getPartition(Amask, PartitionManager::part::B), itp);
 }
 
+struct TestMainSolver : MainSolver {
```

Comment:
I don't particularly like the approach of creating subclasses solely for the purpose being able to access some protected member.

I think a better solution here would be to do what `LRAInterpolationTest` is doing, i.e., explicitly track the partitions in each test.
That way the public API is sufficient.